metrics/deploy/lib/deploy/deploy_command.dart

```diff
@@ -0,0 +1,94 @@
+import 'package:args/command_runner.dart';
+import 'package:process_run/process_run.dart' as cmd;
+import 'package:process_run/shell.dart';
+import 'package:deploy/cli/firebase/firebase_command.dart';
+import 'package:deploy/cli/gcloud/gcloud_command.dart';
+import 'package:deploy/cli/git/git_command.dart';
+
+/// class extending [Command] to facilitate  building GCloud project,build and deploy metrics app.
+class DeployCommand extends Command {
+  @override
+  final name = "deploy";
+  @override
+  final description =
+      "Creates GCloud and Firebase project and deploy metrics app.";
+
+  final firebase = FirebaseCommand();
+  final gcloud = GCloudCommand();
+  final git = GitCommand();
```

Comment:
Can we make these fields private?
```suggestion
  final _firebase = FirebaseCommand();
  final _gcloud = GCloudCommand();
  final _git = GitCommand();
```